controllers/nodehealthcheck_controller.go

```diff
@@ -852,3 +845,12 @@ func getCondition(remediationCR *unstructured.Unstructured, conditionType string
 	}
 	return nil
 }
+
```

Comment:
NIT: I think a function declaration would be useful here, as it is not straightforward what it does. 

IIUC _updateRequeueAfter_ function sets the `result.RequeueAfter` as the minimum duration of the current duration,_RequeueAfter_, and the other one, _newRequeueAfter_. Can you add this to the function declaration?

Reply:
> it is not straightforward what it does

not? 🤔 

however, done